.github/workflows/KubernetesExternalClients.yaml

```diff
@@ -117,12 +129,6 @@ jobs:
     - name: Set output of external-ip
       id: external-ip
       run: echo "::set-output name=external-ip::${{ env.EXTERNAL_IP }}"
-    - name: Set output of gke_zone  
```

Comment:
what was the reason to change it to earlier steps?

Reply:
I changed this because once I canceled the workflow, and the kubernetes cluster is not shut down due to clusterName not being set (it was being set as the last step). I changed it so that it's set right after the cluster is created. 

Cleanup step needs cluster name to clean clusters.